packages/core/src/tracing/trace.ts

```diff
@@ -238,12 +239,17 @@ function createChildSpanOrTransaction(
   return parentSpan ? parentSpan.startChild(ctx) : hub.startTransaction(ctx);
 }
 
-function normalizeContext(context: TransactionContext): TransactionContext {
+function normalizeContext(context: StartSpanOptions): TransactionContext {
```

Comment:
Can we add a small comment explaining what is normalized and why things need normalization?